app/backend/src/couchers/servicers/pages.py

```diff
@@ -16,8 +16,8 @@ def CreatePage(self, request: pages_pb2.CreatePageReq, context):
     def EditPage(self, request: pages_pb2.EditPageReq, context):
         with session_scope() as session:
             page = session.query(Page).get(request.page_id)
-            if page.can_edit(context.user_id):
+            try:
                 page.edit(context.user_id, **request)
-            else:
+            except:
```

Comment:
Please state which exception you catch, instead of caching all errors including KeyboardInterrupt.

Reply:
Yes, I will do. Maybe we need to define some special exception(s)